src/v2/System/Server/sharifyHelpers.ts

```diff
@@ -64,7 +67,7 @@ function generateCustomSharifyScript(sharifyData) {
 
 export function getV2SharifyScript(sharifyData) {
   const script = generateCustomSharifyScript(
-    pick(sharifyData, V2_SHARIFY_ALLOWLIST)
+    pick(sharifyData, [...V2_SHARIFY_ALLOWLIST])
```

Comment:
why spread here?

Reply:
Since I changed it to a `const`, the lodash type complaining and not accepting me casting it `as string[]` for whatever reason.